staging/src/k8s.io/dynamic-resource-allocation/structured/schedulerapi/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS file documentation: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - kubernetes/sig-autoscaling-reviewers
+  - kubernetes/wg-device-management
```

Comment:
There is no `wg-device-management` in any aliases definition in k/k (that I can tell). The common pattern as of now is @pohly and @klueska for WG Device Management approvers